prometheus-metrics/src/main/scala/org/http4s/metrics/prometheus/Prometheus.scala

```diff
@@ -78,18 +97,44 @@ object Prometheus {
   def collectorRegistry[F[_]](implicit F: Sync[F]): Resource[F, CollectorRegistry] =
     Resource.make(F.delay(new CollectorRegistry()))(cr => F.delay(cr.clear()))
 
-  /** Creates a  [[MetricsOps]] that supports Prometheus metrics
-    * *
-    * * @param registry a metrics collector registry
-    * * @param prefix a prefix that will be added to all metrics
+  /** Creates a [[MetricsOps]] that supports Prometheus metrics
+    *
+    * @param registry a metrics collector registry
+    * @param prefix a prefix that will be added to all metrics
+    * @param metricsNames names of the metrics that will be used as a suffix for the metrics
     */
   def metricsOps[F[_]: Sync](
```

Comment:
Could we add a new method which takes the `metricNames` parameter and then have this method's signature remain unchanged? Something like,

```scala
def metricsOps[F[_]: Sync](
      registry: CollectorRegistry,
      prefix: String = "org_http4s_server",
      responseDurationSecondsHistogramBuckets: NonEmptyList[Double] =
        DefaultSettings.responseDurationSecondsHistogramBuckets
  ): Resource[F, MetricsOps[F]] =
    metricsOps[F](registry, prefix, responseDurationSecondsHistogramBuckets, DefaultSettings.metricsNames)
```

This would avoid a binary breaking change, which is always nice, even on a pre-releas.

Reply:
Good idea, gonna change it.

Reply:
Uh, I've remembered one use case - it's might be useful to have a method that specifies prefix of metrics' names and provides custom suffixes (that could be created once for a bunch of endpoints and reused).

Reply:
> Uh, I've remembered one use case - it's might be useful to have a method that specifies prefix of metrics' names and provides custom suffixes (that could be created once for a bunch of endpoints and reused).

I'm sorry, I don't quite follow how this relates to keeping the existing signature unchanged. Could you elaborate?